app/routes.py

```diff
@@ -95,3 +95,40 @@ def get_id(kml_admin_id):
         ),
         200
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['DELETE'])
+def delete_id(kml_admin_id):
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    item = enforcer.get_item(kml_admin_id)
+    file_id = item['file_id']
```

Comment:
This line needs to be moved below line 114, otherwise it will raise an error in case item is None.